cmd/frontend/graphqlbackend/site_alerts.go

```diff
@@ -67,6 +67,12 @@ func init() {
 		}
 		return problems
 	})
+	conf.ContributeWarning(func(c conf.Unified) (problems conf.Problems) {
+		if globals.ExternalURL().Scheme == "http" {
+			problems = append(problems, conf.NewCriticalProblem("Your connection is not private. We recommend using HTTPS connection to access the website. Read more about that [**here**](/help/admin/nginx)"))
```

Comment:
Suggested wording change and also this should be NewSiteProblem because this is in our site config, not critical config. Do you agree @slimsag ?

```suggestion
			problems = append(problems, conf.NewSiteProblem("Your connection is not private. We recommend [configuring Sourcegraph to use HTTPS/SSL](https://docs.sourcegraph.com/admin/nginx)"))
```

Reply:
Yes, we don't have critical config anymore. The critical problem type should in fact be removed.